### ISSUE TITLE
Add export option "CSV Dive sites sorted by Country" and change geo nodes in dive sites XML

### ISSUE THREAD

**Issue by hwillemen** [Subsurface-divelog/subsurface#4140]
### Describe the issue:

- [ ] Bug
- [X] Change request
- [X] New feature request
- [ ] Discussion request

### Issue long description:

Because the way the Dive sites xml is structured it is hard to parse in external programs.
As an example: there is no schema that can import the xml into Excel because of the non-unique "geo" tag.

Solution 1: CHANGE xml export:
Since every geo cat has a specific meaning it would be more readable and better parse-able to make them all unique:
  <geo cat='1' -> <Ocean value='Foo'/>
  <geo cat='2' -> <Country value='Foo'/>
  <geo cat='3' -> <State value='Foo'/>
  <geo cat='4' -> <County value='Foo'/>
  <geo cat='5' -> <Town value='Foo'/>
  <geo cat='6' -> <City value='Foo'/>
(For clarity the format geo1ocean might even be better)
It would also help to give the option to sort the dive sites by Country and then by site name.
This makes more sense than sorting by uuid.

Solution 2: ADD export option for sorted CSV:
Properties like; uuid, name, gps, description, Ocean, Country, State, County, Town, City, Notes
can easily be columns for an CSV.
Again, this should be sorted by Country and then by name, this makes the most sense I think.
"CSV Dive sites by Country/name" would be descriptive.

### Operating system:

Android & Windows: not relevant

### Subsurface version:

6.0.5112-CICD-release

**Comment by hwillemen** [comment]
I have solved this problem myself for now with a python script:
https://github.com/hwillemen/subsurface-divelogs-scripts/blob/main/Divesites_sorter.py

Best would be to simply support this by the program itself.

**Comment by poehoes** [comment]
Nice, perhaps you could add a small non-sorted subsurface.xml and a sorted .csv to your repository so people can test this (and compare the result with the csv).

**Comment by hwillemen** [comment]
I did, as well with a readme and some other info.
